### DISCUSSION TITLE
StacksStorm Helm Chart Deployment Failing in AKS

### DISCUSSION THREAD

**Discussion by vjnallacheruvu** [StackStorm/st2#5494] [Operations]
Deployment getting failed with log post hooks failure in Azure Kubernetes Cluster. Need help!

**Comment by arm4b** [comment]
Could you provide more info?
Failed post hooks should have some logs, - that'll help to identify the issue.

**Reply by vjnallacheruvu** [reply]
Hi,

I have deployed stackstorm using Helm Charts in AKS, once the deployment is
successful I tried to edit the values(custom) in values.yaml and did helm
upgrade, all the containers are going into CrashLoopBackOff mode.
I'm also getting Post Hooks Install failed as error message. When I
provided --no-hooks flag to helm upgrade still the containers are crashing
and unable to login into the UI. Please advise what to do next.. I can
provide more info if needed
